### ISSUE TITLE
Customizing edges of widget and deleting element

### ISSUE THREAD

**Issue by nck23** [mehdieft95/CoolWebPlayer#4]
Hi I would like to use the widget without edges but there's on the right side "is playing" bars showing I really don't like that, is there way to edit that in config file? Also is there way maybe when turning on without edges I would like to click only on buttons not where the edges were before and are now invisible. I found a bug when on win11 you click on wifi/sound/battery button and spotify widget shows up there, squareplayer gets really laggy and only way to fix it is to close spotify and open again and then its fine but only if you dont click that button again.

![scr1](https://github.com/user-attachments/assets/3310c9e5-908a-46cc-91a9-33280b65787e)

**Comment by Meti0X7CB** [comment]
yes u can change the FrostLevel and CornerType value to None to make the skin transparent
